src/content/documentation/publish/source-code-submission.md

```diff
@@ -99,6 +97,24 @@ You can add source code to an existing add-on version. To do this, open [My Add-
 
 {% capture content %}
 
+### Default reviewer build environment
+Reviewers will try to replicate your build environment if specified. However, if you do not specify the build environment, here's the default build environment that reviewers use:
+
+- Ubuntu 22.04 LTS (Desktop edition)
+- 10GB of system memory (RAM), 6 cores of vCPU
+- Node 20 LTS and npm 10
+- 35GB of free disk space
```

Comment:
In [previous](https://github.com/mozilla/extension-workshop/pull/1758#discussion_r1425550871) [comments](https://github.com/mozilla/extension-workshop/pull/1758#discussion_r1425552660) @wagnerand suggested we include specific version numbers to remove any potential ambiguity. While I agree with the motivation behind this suggestion, I'm hesitant about the maintenance overhead that explicit version numbers creates. 

As an alternative, what if we said something like "The latest Node.js LTS release" and link this to either the [Node.js download page](https://nodejs.org/en) or the documentation on [previous releases](https://nodejs.org/en/about/previous-releases#looking-for-latest-release-of-a-version-branch). Similarly for Ubuntu we could link to the Ubuntu website's [downloads page](https://ubuntu.com/download), [releases page](https://www.releases.ubuntu.com/), or the wiki's [releases page](https://wiki.ubuntu.com/Releases).

Reply:
There is definitely a little bit of overhead involved updating this page. However, we don't plan to re-create the VM very often, mostly for major updates for ubunut and node, maybe python which happen only every few years.
The bigger risk I see is us forgetting to update this page when we re-recreate the VM, which is why I [asked](https://mozilla-hub.atlassian.net/wiki/spaces/FDPDT/pages/435224648/Default+build+environment+for+AMO+reviewers?focusedCommentId=493813858) that we add an "update documentation page" step.

Adding links either way sounds great.